docs/bokeh/source/docs/releases/3.7.0.rst

```diff
@@ -7,3 +7,4 @@ Bokeh version ``3.7.0`` (January 2025) is a minor milestone of Bokeh project.
 
 * Improved ``PanTool``'s cursor and state management handling (:bokeh-pull:`14106`)
 * Improved handling of ``touch-action`` based on active tools and their supported events (:bokeh-pull:`14109`)
+* Allowed ``CrosshairTool`` and ``HoverTool`` to work on axes (:bokeh-pull:`14124`)
```

Comment:
"work on" sort of implies to me that a crosshair or tooltip will pop up on the axis itself. Maybe "trigger from axes" or similar?